apis/websocketdatastructures.go

```diff
@@ -101,6 +101,10 @@ type WebsocketScanCommand struct {
 	//
 	// Example: bcae378eacedab83da66079d9366c8f5df542d7ed9ab23bf487e3e1a8481375d
 	ImageHash string `json:"imageHash"`
+	// InstanceID for relevancy scan
+	// namespace-<namespace>/<kind>-<name>/<resourceVersion>
+	// Example: namespace-default/pod-nginx/75641
+	InstanceID *string `json:"instanceID,omitempty"`
```

Comment:
so we do 1 scan per replica?
in the case of 1 Deployment, with 3 containers and 3 replicas we will have 1*3*3=9 calls ?
can you confirm @dwertent ?

Reply:
The `instanceID` is for the parent wl, in your example, the Deployment